mst/mst-multi-node.md

```diff
@@ -1,10 +1,12 @@
-<highlight type="important">
-Timescale currently offers two hosting options. If you are a Timescale Cloud
-(previously called Timescale Forge) user, please use the documentation for
-[setting up multi-node on Cloud](/cloud/latest/cloud-multi-node) instead.
+# Setting up TimescaleDB 2.0 multi-node on Managed Service for TimescaleDB
+
+<highlight type="warning">
+This section describes a feature that is deprecated on Managed Service for
+TimescaleDB. We strongly recommend that you do not use this feature in a
+production environment. If you need more information, please  
```

Comment:
we could suggest users to use TS Cloud for multi-node.
"If you need multi-node, you may check [Timescale Cloud](https://docs.timescale.com/timescaledb/latest/overview/deployment-options/#managed-service-for-timescaledb), and for more information, please contact the support team."